code/modules/vending/restock_tracking/restock_tracker.dm

```diff
@@ -0,0 +1,41 @@
+#define LOW_STOCK_THRESHOLD 75
+
+/**
+ * The restock tracker computer keeps tabs on which vending machines on station are running low on stock.
+ * It can be used by cargo technicians to quickly find out which machines need to be restocked, and rewards crew
+ */
+
+/obj/machinery/computer/restock_tracker
```

Comment:
i remember us talking about making this an ntos app, anything you ran into that prevented that?

Reply:
My poor memory